### DISCUSSION TITLE
榕典与俗语大全之小品文Further Reading

### DISCUSSION THREAD

**Discussion by Guanchishan** [MindongLab/yngdieng#983] [冇讲 (杂谈)]
一个比较感性的提议。同时适用于榕典与俗语大全，但是由于没找到榕典的repo，就先在这里提出issue。
过去没有在线词典，很多词汇我都是拖进百度、谷歌，而后查看结果，来确定其用法、写法、读音的。在这样的过程中，我发现了很多人——老人居多——留下了不少和福州话词汇与俗语有关的小品文。试举几例：
- 罔去🔗一只特立独行的rabbit. [“蛮去”的生活](https://www.jianshu.com/p/594699b5ee7c). 简书.
- 乌黜鬼🔗老猫侠yz. [人生之书——马彩文老师](http://blog.sina.com.cn/s/blog_5d7ddd390100d162.html). 新浪博客.
- 干杯🔗陈希我. [何谓边缘生活――以福州为例](http://sports.cctv.com/20080509/106905.shtml).
- 福州三中朱天骐的微博及博文，此处不枚举。

这类小品文，严格来讲并没有对词汇下什么定义，对俗语做什么考证，完全是词典学之外的事情。但是这类小品文，凝结了作者自己对这些语汇的感触，连结了自己的过往与眼中的当时社会，既有口述小史的作用，又帮助人对从语汇到地方文化都更有切面的感性的认识。
所以如果能在具体的语汇条目页面的下方或侧边栏提供一个Further Reading框，链接到载有这类小品文的网页，或许也能令到词汇解释页更加具有烟火气？
相信这个功能的难点不在技术，而更在文章链接的搜罗。当然该功能对内容并无完备的要求，只需随见随录即可，所以一待有暇，似乎可以随时上线。
不过该功能若上线，需要对用户警告有关链接为外部链接，不为其安全性负责；作者观点亦与本站无关。外链的收录需要有人把关（我很有兴趣），同时开放邮件提供文章链接。文章内容可能要私下存档一遍，以防原站点不可用。
有的文章不一定是小品文，而是对语汇乃至民俗的详细辩正。这种文章或许Further Reading的价值，但要和上述小品文分割。也有像[某些内容农场有收录类似文章](https://fengsu.supfree.net/up.asp?id=17333)，则根本不用予其曝光机会。对其事实进行查核后通过重写的方式吸收入现有词典、语典数据即可。
方才和母语为宁德话的朋友分享完该想法后，他甚至提出可以接受漫谈小品的投稿。不过这或许就是后话了。
以上。敬请参酌。

**Comment by Guanchishan** [comment]
先建了一个库：https://github.com/Guanchishan/Storia-di-Hokchew

另外一个题外话，cdo-corpus的所有权已移交给MindongLab，access给了内容编辑小组。

**Comment by ztl8702** [comment]
不错的主意。

说到“内容农场”。我早前也想做福州话/闽东语甚至所有汉语的网络资源的“内容农场”。https://github.com/MindongLab/meta/issues/1 像 [Hackernews](http://news.ycombinator.com/) 那样可以众人投稿、评论，内容质量或依靠网友评分+某种机器评分筛选。目的也和你说的有点像：搜录除了词典、正经文本（语料库）所需数据之外，其他所有的资源（包括福州话相关微信公众号、语保资讯等）。这样这些东西不会丢掉，至少有个链接和镜像可供查询。

当然那个野心太大，工程很大。暂且不做。

短期而言，你提到的方案，和 https://github.com/Guanchishan/Storia-di-Hokchew 里面的格式（按照frontmatter里面的keyword来和榕典匹配？），很可行。

> 另外一个题外话，cdo-corpus的所有权已移交给MindongLab，access给了内容编辑小组。

收到，谢谢。这个cdo-corpus你下一步准备做什么？

**Comment by Guanchishan** [comment]
> 按照frontmatter里面的keyword来和榕典匹配

就是如许匹配，无赚。我还有一个mentioned字段，解释下，文章只和报菜名一样提到了这个词，没展开，更没有围绕这个词发生故事，就是mentioned。
围绕这个词发生了故事，或者浅谈了其文化意义的，才叫keyword。
这可以充作一个简单的权重判断。很显然，指定文章和keyword的距离更近，在词条解释页要更优先排列。而mentioned关联性就不甚强了。

corpus下一步做什么？首先闽剧字幕还是先在里面更新。其次我有弄到什么语料还是放在里面，这个库可能短期内不能直接干出什么大事，但是就作为一个可以随时取用的库吧。以及未来要是有想做语料库，也就用这个直接做起来。
